src/lesson6.java

```diff
@@ -0,0 +1,33 @@
+import java.util.concurrent.Callable;
+
+public class lesson6 {
+    public static class Cat extends Animals{
+        public Cat (int RunLimit, int SmimLimit){
+            super(RunLimit, SmimLimit);
+        }
+        @Override
+        void SetName(String Name) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.